src/schema/users.ts

```diff
@@ -1044,6 +1078,55 @@ export const resolvers: IResolvers<any, Context> = {
 
       return { _: true };
     },
+    acceptFeatureInvite: async (
+      _,
+      args: {
```

Comment:
Why do you parse it under `args` you can actually directly add the object with params, it's neater.
Maybe args was some legacy thing inside here.